.github/workflows/release-automation.yaml

```diff
@@ -0,0 +1,33 @@
+name: Release Finch latest version
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
Why do we need a release for every push on the main branch?

Reply:
I think this is for every new tag on main branch